SafeExamBrowser.UserInterface.Desktop/Windows/Taskbar.xaml.cs

```diff
@@ -91,7 +91,10 @@ public void Focus(bool forward)
 
 				if (forward)
 				{
-					SetFocusWithin(ApplicationStackPanel.Children[0]);
+					if (ApplicationStackPanel.Children.Count > 0)
```

Comment:
Wouldn't it be better to combine the two conditions in one singular if-statement and otherwise focus the quit button?

Reply:
You're right, that would most likely be better. Done.

Reply:
Excellent, thanks a lot.